app/src/common/constants/retentionPolicy.js

```diff
@@ -17,4 +17,5 @@
 export const RETENTION_POLICY = {
   REGULAR: 'REGULAR',
   IMPORTANT: 'IMPORTANT',
+  RETENTION_POLICY_KEY: 'retentionPolicy',
```

Comment:
Let's store it as a separate constant as it is not one of the 'retention policy types'.